.circleci/config.yml

```diff
@@ -1,61 +1,39 @@
 # Ruby CircleCI 2.0 configuration file
 # Check https://circleci.com/docs/2.0/language-ruby/ for more details
-version: 2
+version: 2.1
+orbs:
+  browser-tools: circleci/browser-tools@1.4.8
+  node: circleci/node@5.2.0
+  ruby: circleci/ruby@2.1.2
 jobs:
   build:
     docker:
       # Specify the Ruby version you desire here
-      - image: cimg/ruby:3.1-browsers
-
-    working_directory: ~/identity-dev-docs
+      - image: cimg/ruby:3.2-browsers
+    environment:
+      BUNDLE_JOBS: '3'
+      BUNDLE_RETRY: '3'
     steps:
-      - checkout
-      
-      - run:
-          name: Switch Node.js version
-          command: |
-            wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash
-            export NVM_DIR="$HOME/.nvm"
-            . "$NVM_DIR/nvm.sh" --install --latest-npm
-            echo 'export NVM_DIR="$HOME/.nvm"' >> $BASH_ENV;
-            echo '[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"' >> $BASH_ENV;
-      
+      - node/install
```

Comment:
Neat, I didn't know CircleCI had [built-in support for NVM](https://circleci.com/developer/orbs/orb/circleci/node#commands-install)! That feels a lot nicer / safer than running code directly from GitHub 😅 I wonder if that's always been a thing, or if it's a new feature.

Reply:
I think this feature is relatively new. I forget when I first discovered it. Maybe in the past 2 years? It looks like the Node Orb was originally created in 2018, but I'm not sure if it had all of today's functionality back then.